elements/pfe-sass/extends/_extends.scss

```
@@ -57,45 +57,45 @@
   }
 }
 
-%ordered-list-primary {
-  // size of the font inside of the circle
-  --FontSize: var(--pfe-list-FontSize, 24px);
-  // size of the space between the circle and the content
-  --gap: var(--pfe-list--gap, 24px);
-  // amount of space between list items
-  --margin: var(--pfe-list--margin, 24px);
-  // font color of the numbers in the circle
-  --Color: var(--pfe-list--Color, rgba(238, 0, 0, 1));
-  // background color of circle
-  --BackgroundColor: var(--pfe-list--BackgroundColor, rgba(238, 0, 0, 0.07));
+// Mixins and extends can't use the LOCAL-VARIABLES map because we would be
+// at risk of overriding an elements map if they call our mixin. We need to
+// use just regular sass variables and pfe-local and specify the $fallbacks.
+@mixin ordered-list-primary {
```

```suggestion
@mixin ordered-list-primary($size: 56px) {
```

You can move the $size up to an optional input; that seems like the most likely candidate for customization.